Submodule linux-kernel-4.17 contains modified content
diff --git a/linux-kernel-4.17/Makefile b/linux-kernel-4.17/Makefile
index 32c83a163544..30556c2dce32 100644
--- a/linux-kernel-4.17/Makefile
+++ b/linux-kernel-4.17/Makefile
@@ -1787,3 +1787,10 @@ FORCE:
 # Declare the contents of the .PHONY variable as phony.  We keep that
 # information in a variable so we can use it in if_changed and friends.
 .PHONY: $(PHONY)
+
+#################
+# Adiciones lab #
+#################
+
+core-y += ../practica_lab
+
diff --git a/linux-kernel-4.17/arch/x86/entry/syscalls/syscall_64.tbl b/linux-kernel-4.17/arch/x86/entry/syscalls/syscall_64.tbl
index 4dfe42666d0c..29af2c42a014 100644
--- a/linux-kernel-4.17/arch/x86/entry/syscalls/syscall_64.tbl
+++ b/linux-kernel-4.17/arch/x86/entry/syscalls/syscall_64.tbl
@@ -342,6 +342,14 @@
 331	common	pkey_free		__x64_sys_pkey_free
 332	common	statx			__x64_sys_statx
 
+#################
+# Adiciones lab #
+#################
+333	64	ssoo_custom_syscall	__x64_sys_saying
+#################
+# Contin. orig. #
+#################
+
 #
 # x32-specific system call numbers start at 512 to avoid cache impact
 # for native 64-bit operation. The __x32_compat_sys stubs are created
diff --git a/linux-kernel-4.17/include/linux/syscalls.h b/linux-kernel-4.17/include/linux/syscalls.h
index 70fcda1a9049..d7a9c2d421b1 100644
--- a/linux-kernel-4.17/include/linux/syscalls.h
+++ b/linux-kernel-4.17/include/linux/syscalls.h
@@ -1268,4 +1268,12 @@ static inline long ksys_truncate(const char __user *pathname, loff_t length)
 	return do_sys_truncate(pathname, length);
 }
 
+// #################
+// # Adiciones lab #
+// #################
+asmlinkage long sys_saying(void);
+// #################
+// # Contin. orig. #
+// #################
+
 #endif
