diff --git a/Makefile b/Makefile
index 32c83a1..1cefd2b 100644
--- a/Makefile
+++ b/Makefile
@@ -980,6 +980,14 @@ endif
 ifeq ($(KBUILD_EXTMOD),)
 core-y		+= kernel/ certs/ mm/ fs/ ipc/ security/ crypto/ block/
 
+#################
+# Adiciones lab #
+#################
+core-y		+= ../practica_lab/
+#################
+# Contin. orig. #
+#################
+
 vmlinux-dirs	:= $(patsubst %/,%,$(filter %/, $(init-y) $(init-m) \
 		     $(core-y) $(core-m) $(drivers-y) $(drivers-m) \
 		     $(net-y) $(net-m) $(libs-y) $(libs-m) $(virt-y)))
diff --git a/arch/x86/entry/syscalls/syscall_64.tbl b/arch/x86/entry/syscalls/syscall_64.tbl
index 4dfe426..ca91c3b 100644
--- a/arch/x86/entry/syscalls/syscall_64.tbl
+++ b/arch/x86/entry/syscalls/syscall_64.tbl
@@ -342,6 +342,14 @@
 331	common	pkey_free		__x64_sys_pkey_free
 332	common	statx			__x64_sys_statx
 
+#################
+# Adiciones lab #
+#################
+333	64	ssoo_custom_syscall	sys_saying
+#################
+# Contin. orig. #
+#################
+
 #
 # x32-specific system call numbers start at 512 to avoid cache impact
 # for native 64-bit operation. The __x32_compat_sys stubs are created
diff --git a/include/linux/syscalls.h b/include/linux/syscalls.h
index 70fcda1..4d18782 100644
--- a/include/linux/syscalls.h
+++ b/include/linux/syscalls.h
@@ -1268,4 +1268,12 @@ static inline long ksys_truncate(const char __user *pathname, loff_t length)
 	return do_sys_truncate(pathname, length);
 }
 
+// #################
+// # Adiciones lab #
+// #################
+asmlinkage long sys_saying(int, int);
+// #################
+// # Contin. orig. #
+// #################
+
 #endif
